hypervisor/arch/x86/ept.c

```diff
@@ -103,19 +103,12 @@ uint64_t gpa2hpa(struct vm *vm, uint64_t gpa)
 	return local_gpa2hpa(vm, gpa, NULL);
 }
 
+/**
+ * @pre: only SOS will use hpa2gpa and they're identical mapping in SOS.
+ */
```

Comment:
given that we only support SOS usage, how about to change the API to sos_hpa2gpa?
and make sure the caller will check the right parameter. Then we don;'t need to check the parameter.

Reply:
It could be. But it's hard to implement.
Now the code logic is like this
#if not partition mode
if (is_vm0)
    call init_vm_boot_info (which will called hpa2gpa)
#else
    init_vm_boot_info (for partition mode which will not call hpa2gpa)
#endif